packages/googlecloud/functions/getanswer/inquirer.py

```diff
@@ -192,18 +159,65 @@ def get_indepth_response_from_query(df, db, query, k):
     """
 
     prompt = PromptTemplate(
-        input_variables=["question", "docs"],
-        template=template,
-    )
+            input_variables=["question", "docs"],
+            template=template,
+        )
 
     chain_llm = LLMChain(llm=llm, prompt=prompt)
-    responses_llm = chain_llm.run(question=query, docs=docs_page_content, temperature=1)
-    print(responses_llm)
+    responses_llm = chain_llm.run(
+            question=query, docs=docs_page_content, temperature=0
+        )
+
+    return process_responses_llm(responses_llm, docs, query_type)
+
+## varied responses for user annotation
+def get_varied_response_from_query(df, db, query, k, n = 1, card_type = "varied"):
+    logger.info("Performing varied summary query...")
+    master_response = {}
+    master_response["card_type"] = "varied"
+    response_list = {}
+    k_list = [3,5,7]
```

Comment:
Why suck small k values? Also not a big difference between them? My gut would be 6, 12, 25.